managed/src/main/java/com/yugabyte/yw/models/HealthCheck.java

```diff
@@ -64,7 +91,9 @@ public static HealthCheck addAndPrune(UUID universeUUID, Long customerId, String
     check.idKey = HealthCheckKey.create(universeUUID);
     check.customerId = customerId;
     // Validate it is correct JSON.
-    check.detailsJson = Json.stringify(Json.parse(details));
+    ObjectMapper objMapper = new ObjectMapper();
```

Comment:
Just use `Json.fromJson`

Reply:
Avoid creating new ObjectMapper if you are doing that in any of the diffs in flight.